jack.py

```diff
@@ -1480,6 +1485,63 @@ def get_ports(self, name_pattern="", is_audio=False, is_midi=False,
             _lib.jack_free)
         return self._port_list_from_pointers(names)
 
+    def get_internal_client_name(self, intclient):
+        """Get the name of an internal client given the handle."""
+        client_name = _ffi.gc(_lib.jack_get_internal_client_name(
+            self._ptr, intclient), _lib.jack_free)
+        return _ffi.string(client_name).decode()
+
+    def internal_client_handle(self, client_name):
+        """Get the handle for an internal client name."""
+        status = _ffi.new("jack_status_t*")
+        intclient = _lib.jack_internal_client_handle(self._ptr, client_name.encode(), status)
+        if not intclient:
+            status = str(Status(status[0]))
+            raise JackError("Unable to get handle for {0!r}: {1}".format(client_name, status))
+        return intclient
+
+    def internal_client_load(self, name, use_exact_name=False, load_name=None, load_init=None):
+        """
+        Parameters
+        ----------
+        name : str
+            The desired client name of at most :func:`client_name_size`
+            characters.  The name scope is local to each server.
+            Unless forbidden by the `use_exact_name` option, the server
+            will modify this name to create a unique variant, if needed.
+
+        Other Parameters
+        ----------------
+        use_exact_name : bool
+            Whether an error should be raised if `name` is not unique.
+            See :attr:`Status.name_not_unique`.
+        load_name : str
+            The shared object file from which to load the new internal client
+            (otherwise use `name`)
+        load_init : str:
+            An arbitary string passed to the internal client's
+            jack_initialize() routine (otherwise NULL), of no more than
+            :func:`load_init_limit` bytes.
+        """
+        status = _ffi.new("jack_status_t*")
+        args = [name.encode(), _lib.JackNullOption, status]
```

Comment:
I found that all this `args[1]` stuff is quite confusing, I changed it in `Client.__init__()` in c5982a39b307408e81945c2a4943e5a677e450bd, can you please also change it here?